devtool/devtool.go

```diff
@@ -266,7 +266,8 @@ func (d *DevTools) resolveHost(ctx context.Context) error {
 
 	addrs, err := net.DefaultResolver.LookupHost(ctx, origHost)
 	if err != nil {
-		return err
+		log.Printf("error resolving host %s: %v\n", origHost, err)
+		addrs = append(addrs, origHost)
```

Comment:
I took a quick look at the code and it looks to me like there's possibly one case where addrs is non-nil and err is also non-nil. In this case, just to be sure I think it would be best to not append here. Thoughts?